src/parser/transform/expression/transform_constant.cpp

```diff
@@ -11,6 +11,7 @@ unique_ptr<ParsedExpression> Transformer::TransformValue(PGValue val) {
 		assert(val.val.ival <= numeric_limits<int32_t>::max());
 		return make_unique<ConstantExpression>(SQLType::INTEGER, Value::INTEGER((int32_t)val.val.ival));
 	case T_PGBitString: // FIXME: this should actually convert to BLOB
```

Comment:
This should be removed, right?

Reply:
Indeed, T_PGBitString is for bit string '010101'.